<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoDialog</class>
 <widget class="QDialog" name="VideoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>540</y>
     <width>641</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QSlider" name="shutterSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="uvLabel">
      <property name="text">
       <string>UV:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="gainSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="vrLabel">
      <property name="text">
       <string>VR:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="gainLabel">
      <property name="text">
       <string>Gain:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QSlider" name="uvSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="wbLabel">
      <property name="text">
       <string>White balance =&gt; </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="shutterLabel">
      <property name="text">
       <string>Shutter:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QSlider" name="vrSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="pixmapLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>shutterSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>VideoDialog</receiver>
   <slot>onShutterChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>VideoDialog</receiver>
   <slot>onGainChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uvSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>VideoDialog</receiver>
   <slot>onUVChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vrSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>VideoDialog</receiver>
   <slot>onVRChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onShutterChanged(int)</slot>
  <slot>onGainChanged(int)</slot>
  <slot>onUVChanged(int)</slot>
  <slot>onVRChanged(int)</slot>
 </slots>
</ui>
